### PR TITLE
Add new packages from conda

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: packages.tsv [hunk 1]
```diff
@@ -9,6 +9,8 @@ alpine	jq	1	test $(/usr/bin/jq -M -n -c "{a: (4+3)}") = "{\"a\":7}"
 alpine	mtr	1	/usr/sbin/mtr -v
 conda	m4	1.4.17--0	test $(echo "define(a, bbb)a" | m4) = "bbb"
 conda	samtools	1.3--1	samtools --version
+conda	samtools	1.2--2	samtools --version
```
Comment (thriqon) [inline packages.tsv]: This will fail, since it is a duplicate package. Could you _change_ the `samtool` line instead of adding a new one?

Comment (bgruening) [inline packages.tsv]: This is what I wanted to test, it should end up as separate version, right? You want to have multiple versions of one package.

Comment (bgruening) [inline packages.tsv]: but granted this can be added later ... will remove it

Comment (thriqon) [inline packages.tsv]: I developed it to _keep_ old versions, but only have one `latest`. The `latest` in the Docker repository corresponds to the version in the `packages.tsv`.

Comment (bgruening) [inline packages.tsv]: Let's put this as discussion point for our next meeting, do you have time next week?

Comment (thriqon) [inline packages.tsv]: I'm always available to meet. Do you have time on Monday or Tuesday?

Comment (bgruening) [inline packages.tsv]: Yes, but I will be @ the ZBSA the entire day, you can drop by if you want. Do we want to target a first release next week? I will try to push a few more packages over the weekend.

Comment (thriqon) [inline packages.tsv]: I can drop by when it's most convenient for you. Yes, let's do that. I'll add more polish here and there to make it presentable :wink:

**Pull request merged**